### PR TITLE
ReviewFile

### PR DESCRIPTION
**Pull request by Domnicdev**
Hi Shaji Kindly Review my project .

### CODE REVIEW

File: ReviewFile
Comment (shajiBS24) [inline ReviewFile:1]: Hi Domnic, The name of insert and update method is not appropriate in DMScontroller.java Kindly, change it.

**Review by shajiBS24 — COMMENTED** [review]
Hi Domnic, The name of insert and update method is not appropriate in DMScontroller.java Kindly, change it.

**Comment by shajiBS24** [comment]
Don't forgot change it in the JUnit file also...

**Comment by Domnicdev** [comment]
Yes Shaji I updated in that file also

**Comment by shajiBS24** [comment]
Other files are fine!

**Comment by Domnicdev** [comment]
Thanks Shaji

**Pull request merged**